powerhome/playbook
# [PLAY-422] Passphrase kit dependency and documentation

#### Screens

![Screen Shot 2022-10-19 at 11 14 19 AM](https://user-images.githubusercontent.com/8194056/196732338-b977c4a2-5788-4a1a-88b6-8aab0ac1ed54.png)

#### Breaking Changes

No breaking changes, removed the `zxcvbn` library from `playbook` and added it only to the `playbook-website`. 

Instead of adding a new prop for `strenght` checking, I added an example showing how to do that with `zxcvbn` using the `passphrase` kit.

#### Runway Ticket URL

[[PLAY-422]](https://nitro.powerhrg.com/runway/backlog_items/PLAY-422) 

#### How to test this

Open milano build and check if the kit renders correctly. 
Check for any console errors.

#### Checklist:
